htdocs/feedback_mri_popup.php

```diff
@@ -161,3 +161,6 @@
 $smarty->display('feedback_mri_popup.tpl');
 
 ob_end_flush();
+
+exit;
```

Comment:
should this be a part of this PR?

Reply:
No I don't think so. I think this happened as the result of a rebase or something. I'll fix it.